setup.cfg

```diff
@@ -8,4 +8,4 @@ universal = 1
 test=pytest
 
 [tool:pytest]
-addopts=--pyarg schema_salad
+addopts=--pyarg -rsxX -q schema_salad/tests/test_cli_args.py schema_salad/tests/test_ref_resolver.py
```

Comment:
Yes, please revert this. We can find another way to report which tests pass and which are still failing